### PR TITLE
Added ArcBrace subclass

### PR DESCRIPTION
**Pull request by Iced-Tea3**
## Motivation

Making the brace follow an arc isn't immediately clear from the documentation.

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [ ] I have written a descriptive PR title (see top of PR template for examples)
- [ ] I have written a changelog entry for the PR or deem it unnecessary
- [ ] My new functions/classes either have a docstring or are private
- [ ] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

**Comment by AntonBallmaier** [comment]
Quick reference: https://manimce--1496.org.readthedocs.build/en/1496/reference/manim.mobject.svg.brace.Brace.html?highlight=arc_brace

Looks great! Maybe you should increase the size of the content a bit :)

File: manim/mobject/svg/brace.py [hunk 1]
```diff
@@ -49,6 +49,16 @@ def construct(self):
                     self.add(br)
                 VGroup(*self.mobjects).arrange(DOWN, buff=0.2)
 
+    .. manim:: ArcBraceExample
+        :save_last_frame:
```
Comment (AntonBallmaier) [inline manim/mobject/svg/brace.py:53]:
```suggestion
        :save_last_frame:
        :ref_methods: Mobject.apply_complex_function
```

**Review by AntonBallmaier — CHANGES_REQUESTED** [review]
Overall I like the addition. However, I have some requests to enhance the example.

**Comment by Iced-Tea3** [comment]
Thank you for the comments! I'll go ahead and fix them

**Comment by Iced-Tea3** [comment]
I'm actually working on the subclass! Might make another PR, or not. This is my first. Can I?

**Comment by AntonBallmaier** [comment]
> I'm actually working on the subclass! Might make another PR, or not. This is my first. Can I?

That is really nice! Generally you can always create Pull Requests for any change to Manim you might find useful. However, for many changes it might be useful to discuss them before actually implementing them - otherwise someone might put a lot of effort into a feature that doesn't get merged in the end. Therefor I would advise you to create an Issue explaining the new feature you would like to add in detail. I know, this sounds like a lot of work (overhead). But actually this can be a great time saver, since it forces you to think about your new feature in detail which makes implementing it a lot easier. On top of that you can often reuse large parts of the Issue in the documentation of you feature :)

**Comment by Iced-Tea3** [comment]
Ouh okay,, thank you! i don't know all the procedures, so that was really helpful.

**Comment by AntonBallmaier** [comment]
As long as you haven't finished your PR you can mark it as a draft so that reviewers know that we should wait until you are done with this PR :slightly_smiling_face:

**Comment by Iced-Tea3** [comment]
Ouh i see, sorry for the delay, but I'll close this pr for now and open an issue since there's quite a lot of things i'm not familiar with, like tests and others. 😅 Thanks again!

**Pull request closed without merging**